### ISSUE TITLE
Authentication with GitHub does not work

### ISSUE THREAD

**Issue by Rose9111** [jitsi/jitsi-meet#14685]
It is not possible to create a room with authentication via GitHub in browser and mobile client (Android) on [meet.jit.si](https://meet.jit.si/)
After authentication is verified, no redirection to the room is made.

**Comment by Rose9111** [comment]
On Android I use the mobile client and auth by Firefox. On Windows - Chrome, Firefox. I don't add any parameters to the URL myself

**Comment by saghul** [comment]
After authenticating, do you just get a black screen? What happens if you pull to refresh?

**Comment by Rose9111** [comment]
Yes - black screen in browser on page: https://web-cdn.jitsi.net/auth-static/meet-jit-si/v10/signin.html with parameters.
Refreshing the page doesn't change anything.

**Comment by saghul** [comment]
@damencho Any ideas?

**Comment by damencho** [comment]
Any errors in the ja console on that page? 
I have tested github yesterday and there was no problem. Have you used other accounts before that use the same email?

**Comment by Rose9111** [comment]
> Any errors in the ja console on that page?

No.
> Have you used other accounts before that use the same email?

I use a private session to prevent other accounts from interfering and authorize only by GitHub. However, when using auth by Gmail , authentication succeeds

**Comment by damencho** [comment]
And do you have the same email in github and gmail?

**Comment by damencho** [comment]
If this is the case this is not supported. You can have only one login with an email.

**Comment by Rose9111** [comment]
I understand it. So that's why I actually log out of Gmail  by using a private session. If I'm not logged in to Gmail, does it really matter whether I have a login there or not?

**Comment by damencho** [comment]
It doesn't matter whether you are logged in or logged out, matters whether you have used it once for meet.jit.si

**Comment by damencho** [comment]
Please, when you have questions or problems use the community forum before opening new issues, thank you.

**Comment by Rose9111** [comment]
I understand. thanks for the help
